src/ui/components/ObjectView.vue

```diff
@@ -21,17 +21,12 @@
 -->
 <template>
   <div>
-    <div
-      v-if="supportsIndependentTime"
+    <!-- <div
```

Comment:
Let's remove this code if it's no longer needed.

Reply:
done!